modules/nodes-cluster-features-about.adoc

```diff
@@ -1,37 +0,0 @@
-// Module included in the following assemblies:
-//
-// * nodes/nodes-cluster-enabling-features.adoc
-
-[id="nodes-cluster-features-about_{context}"]
-= Understanding feature gates and Technology Preview features
-
-You can use the `FeatureGate` custom resource to enable Technology Preview
-features throughout your cluster. This allows you, for example, to enable
-Technology Preview features on test clusters where you can fully test them while
-ensuring they are disabled on production clusters.
-
-[IMPORTANT]
```

Comment:
I don't think we want to remove this.

Reply:
@sferich888 I renamed this module for consistency. This sentence is still in the docs: https://github.com/openshift/openshift-docs/pull/24132/files#diff-17f46d9305040639ceb3dfbc0f5bc64625da1d3b3174750c4b2f377abaf90bb5R10